orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/pipeline/ManualJudgmentStage.groovy

```diff
@@ -73,14 +75,20 @@ class ManualJudgmentStage implements StageDefinitionBuilder, AuthenticatedStage
     final long backoffPeriod = 15000
     final long timeout = TimeUnit.DAYS.toMillis(3)
 
+    @Value('${spinnaker.manual-judgment-navigation:false}')
```

Comment:
I believe there's a bit of prior art under the `stages` prefix.

Perhaps, `stages.manual-judgment.navigation`.